src/SDKs/SqlManagement/Management.Sql/Microsoft.Azure.Management.Sql.csproj

```diff
@@ -7,19 +7,12 @@
 		<PackageId>Microsoft.Azure.Management.Sql</PackageId>
 		<Description>Azure SQL Management SDK library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Sql</AssemblyName>
-		<Version>1.15.0-preview</Version>
+		<Version>1.16.0-preview</Version>
 		<PackageTags>Microsoft Azure SQL Management;SQL;SQL Management;</PackageTags>
 		<PackageReleaseNotes>
 			<![CDATA[
 New features:
-- Database and ElasticPool now use Sku property for scale and tier-related properties. We have made this change in order to allow future support of autoscale, and to allow for new vCore-based editions.
-    * Database.Sku has replaced Database.RequestedServiceObjectiveName and Database.Edition. Database scale can be set by setting Sku.Name to the requested service objective name (e.g. S0, P1, or GP_Gen4_1), or by setting Sku.Name to the sku name (e.g. Standard, Premium, or GP_Gen1) and set Sku.Capacity to the scale measured in DTU or vCores.
-    * Database.CurrentSku has replaced Database.ServiceLevelObjetive.
-    * Database.CurrentServiceObjectiveId and Database.RequestedServiceObjectiveId have been removed.
-    * ElasticPool.Sku has replaced ElasticPool.Dtu. Elastic pool scale can be set by setting Sku.Name to the requested sku name (e.g. StandardPool, PremiumPool) and setting Sku.Capacity to the scale measured in DTU or vCores.
-    * ElasticPool.PerDatabaseSettings has replaced ElasticPool.DatabaseDtuMin and ElasticPool.DatabaseDtuMax.
- - Database.MaxSizeBytes is now a long instead of string.
- - LocationCapabilities tree has been changed in order to support capabilities of new vCore-based database and elastic pool editions.
+- Adding support for SQL Database Agent
```

Comment:
@jaredmoo Is this new features description too plain? Let me know if you'd like a more detailed one. 
I was unsure how detailed to go for a release update.

Reply:
That is totally fine. Maybe also mention the type names (JobAgent, Job, JobStep, ....) so that it is obvious which types are related to DB agent?

Reply:
Will update.